docs/reference/sql-mapper/entities/timestamps.md

```diff
@@ -0,0 +1,37 @@
+# Timestamps
+
+Timestamps can be used to automatically set the `created_at` and `updated_at` fields on your entities.
+
+Timestamps are enabled by default, but can be disabled by setting the `autoTimestamps` option to `false` in configuration.
+
+## Configuration
+
+To enable timestamps, you need to set the `autoTimestamps` field to `true` in configuration file:
```

Comment:
These two sentences contradict themselves.

Reply:
I changed the sentence on how to disable the timestamps